docs/env.rst

```diff
@@ -102,6 +102,22 @@ To know if your Plone instance is running in a test runner, use :meth:`api.env.t
         pass  # do something
 
 
+.. _env_read_only_mode_example:
+
+Read-Only mode
+==============
+
+To know if your Zope / Plone instance is running on a read-only ZODB connection use :meth:`api.env.read_only_mode`.
```

Comment:
Hi @zupo, my use case is on ZRS or RelStorage, when you have an active replication, where the Replicas are read only and the master goes down.

I do use it to allow my Plone instance to run and disable create, delete and update methods in the UI. One Example: https://github.com/lmu/lmu.contenttypes.blog/blob/master/src/lmu/contenttypes/blog/browser/templates/listing_view.pt#L55

I was asked to contribute this in plone.api, after showing that at a sprint.